src/services/SyncService.js

```diff
@@ -270,6 +271,29 @@ class SyncService {
 		})
 	}
 
+	async sendRemainingSteps(queue) {
+		if (queue.length === 0) {
+			return
+		}
+		let outbox = []
+		const steps = queue.map(s => encodeArrayBuffer(s))
+			.filter(s => s < 'AQ')
+		const awareness = queue.map(s => encodeArrayBuffer(s))
+			.findLast(s => s > 'AQ') || ''
```

Comment:
We we have this now in two places with services/WebSocketPolyfill.js maybe we put in a helper function? Could probably then also use a y.js method if available to filter (I'm not a fan of the magic strings there)...

But not a blocker